colcon_installed_package_information/package_augmentation/python.py

```diff
@@ -3,13 +3,17 @@
 
 import sysconfig
 
+try:
+    from importlib.metadata import Distribution
+except ImportError:
+    # TODO: Drop this with Python 3.7 support
```

Comment:
```suggestion
    # TODO: Drop this when dropping Python 3.7 support
```

Minor phrasing suggestion to be clear which direction python 3.7 support is oriented toward here (that we have it and can ditch this when we drop it)